cds/base/templates/helpers/general.html

```diff
@@ -0,0 +1,126 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2012, 2013, 2014, 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% macro get_author(author, is_link=false) %}
+    {% if is_link %}
+      <a href="http://cds.cern.ch/search?f=author&p={{ author.full_name }}&ln=en">
```

Comment:
Could you please use [`url_for`](http://flask.pocoo.org/docs/0.10/api/#flask.url_for) instead?